app/javascript/common/constants.js

```diff
@@ -0,0 +1,42 @@
+export const PostsFilterOptions = [
+  {
+    text: 'Latest Activity',
+    value: 'latest_activity',
+    type: "sort",
+  },
+  {
+    text: 'Most Voted',
+    value: 'most_voted',
+    type: "sort",
+  },
+  {
+    text: 'Most Recent',
+    value: 'most_recent',
+    type: "sort",
+  },
+  {
+    text: '#Open',
```

Comment:
```suggestion
    text: #open,
```

I guess this should be all lowercase